### ISSUE TITLE
Contrast for #wdn-icon-search

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#699]
WCAG2AA
Discovered with: http://squizlabs.github.io/HTML_CodeSniffer/

```
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1:1. Recommendation: change text colour to #fff.
```

Code Snippet:

```
<label class="wdn-icon-search" for="wdn_search_query">Search</label>
```
